tests/generators/OperationGenerator.ts

```diff
@@ -129,7 +129,8 @@ export default class OperationGenerator {
     recoveryPublicKey?: JwkEs256k,
     updatePublicKey?: JwkEs256k,
     otherPublicKeys?: PublicKeyModel[],
-    serviceEndpoints?: ServiceEndpointModel[]) {
```

Comment:
Sanity: Looks like through the recent refactoring, `recoveryPublicKey` and `updatePublicKey` is no longer needed to be passed in?